test/lib/find-scaffolds.js

```diff
@@ -58,22 +63,52 @@ const after = () => {
 };
 
 
-test('Find Scaffolds success', assert => {
+test('Find Scaffolds API', assert => {
+  before();
```

Comment:
Tip: instead of using a generic name like this, you could use a more semantic name like "`stubNpm()`".

Also, as a matter of discipline, you could also localize the `npm` var and inject it as a dependency to findGlobal rather than requiring it. That way you separate the side-effects (npm API I/O) from the program logic, and it allows you to do this:

```js
const npm = stubNpm();

// ... do stuff

resetNpm(npm);
assert.end();
```

I find this easier to read because you can see what's going on inline without scrolling up and looking at the `before()` and `after()` functions. You also avoid the common situation where some tests need more setup/teardown than others, but you run everything for all tests. For example, your first test doesn't need the stub at all (unless I'm reading it wrong), but you're calling the functions anyway...